test/functional/wiki_controller_test.rb

```diff
@@ -319,4 +319,10 @@ def teardown
     end
   end
 
+  test "should get wiki with differnt title and path" do
+    wiki = node(:wiki_page)
+    slug = wiki.path.gsub('/wiki/', '')
+    get :show, id: slug
+    assert_response :success
+  end
 end
```

Comment:
This tests a wiki that is having a different title and path. This will also act as a gatekeeper if we rethink to use this functionality again.